### PR TITLE
Http requests in nodejs

### PR DESCRIPTION
**Pull request by lalith1403**
I have written a descriptive article about HTTP requests and also discuss the various libraries available for making HTTP calls. I request the team to kindly review the same. 

there are two images included with the index.md file. 

author: lalithnarayan c
short_description: Understanding HTTP, HTTP Requests and taking a look at the various libraries that enable the same.

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋  Hi Molly - would you mind giving this Node article a review. 
Trying to prioritize the Node PRs. 🚀 
Thank you 
- Oh - waiting on the author to find out if one image within the article needs a source - or he made it.

**Comment by hectorkambow** [comment]
@lalith1403 
image

**Comment by ninjaginja** [comment]
no problem @hectorkambow ... and by Node, you mean Node.js, right? ;)

**Comment by lalith1403** [comment]
http://wiki.hashphp.org/HttpPrimer... This is the source

File: articles/http-requests-nodejs/index.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+req.write(data)
+req.end()
+```
+
+### fetch
+The fetch module is also built-in and is usually used for making API calls. Fetch predominantly uses the concept of promises. Promises are objects that represent a point in time when the completion or failure of an asynchronous task occurs. Errors in any then statements is caught by the catch statement. Therefore, error handling is much more simplified.
+
+
+```jsx
```
Comment (ninjaginja) [inline articles/http-requests-nodejs/index.md:119]: @lalith1403 The `fetch` code block needs some attention. Looks like the code fencing got a little jumbled, and there's some axios code mixed in there as well.

Comment (lalith1403) [reply articles/http-requests-nodejs/index.md:119]: yikes... I was testing axios code out and had pasted it there for reference. sorry... I am removing those lines right away... the ones that havent been included in the code blocks were for my reference

**Pull request merged**